packages/nuxi/src/commands/module/add.ts

```diff
@@ -305,6 +305,11 @@ async function resolveModule(moduleName: string, cwd: string): Promise<ModuleRes
   // check if a dist-tag exists
   pkgVersion = (pkgDetails['dist-tags']?.[pkgVersion] || pkgVersion) as string
 
+  // if a the major version was provided, search for the latest version that starts with that number
+  if (pkgVersion && pkgVersion.length === 1) {
+    pkgVersion = Object.keys(pkgDetails.versions)?.findLast(version => version.startsWith(pkgVersion as string)) || pkgVersion
```

Comment:
can we use `semver`'s `satisfies` so we support any version range?

Reply:
For sure Dan, working on it

Reply:
Fixed [c734d48](https://github.com/nuxt/cli/pull/803/commits/c734d48482935e3de64cc3e603c1a311344fabf9)